src/OrbitGl/App.cpp

```diff
@@ -338,6 +339,18 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
         main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kInfo,
                                          absl::ZeroDuration(), "Capture started.");
 
+        orbit_version::Version capture_version{capture_started.orbit_version_major(),
+                                               capture_started.orbit_version_minor()};
+        orbit_version::Version current_version = orbit_version::GetVersionNumber();
+        if (capture_version > current_version) {
+          main_window_->AppendToCaptureLog(
+              MainWindowInterface::CaptureLogSeverity::kWarning, absl::ZeroDuration(),
+              absl::StrFormat(
+                  "Capture appears to be coming from the future: it was captured by Orbit version "
+                  "%d.%d, please open the capture using Orbit v%d.%d or above",
+                  capture_version.major_version, capture_version.minor_version,
+                  capture_version.major_version, capture_version.minor_version));
+        }
```

Comment:
Should we also show a message box? This sounds like something the user should be made very well aware of.

Reply:
done.